docs/guides/serving.md

```diff
@@ -0,0 +1,68 @@
+Floyd run command has a [serve mode](../commands/run.md#serve). This will upload the files in the current directory and run a special command - `python app.py`. This file handles the incoming request, executes the logic and returns the output. Floyd expects this file to contain the code to run a web server(based on Flask).
```

Comment:
> This will upload the files in the current directory and run a special command - `python app.py`.

- I understand what this means, but its not clear. We can simplify this further.
- Also provide an example app.py file with the sections that need to be filled in by the user

Reply:
> I understand what this means, but its not clear. We can simplify this further.

Ok, I'll try to improve it.

> Also provide an example app.py file with the sections that need to be filled in by the user

Do you prefer a skeleton/template file or something that runs?